code/src/java/pcgen/gui2/tabs/TabTitle.java

```diff
@@ -33,15 +34,13 @@
 public class TabTitle
 {
 
-	public static final String FOREGROUND = "foreground"; //$NON-NLS-1$
-	public static final String BACKGROUND = "background"; //$NON-NLS-1$
-	public static final String ENABLED = "enabled"; //$NON-NLS-1$
+	static final String ENABLED = "enabled"; //$NON-NLS-1$
 	public static final String TITLE = "title"; //$NON-NLS-1$
-	public static final String ICON = "icon"; //$NON-NLS-1$
-	public static final String TOOLTIP = "tooltip"; //$NON-NLS-1$
+	static final String ICON = "icon"; //$NON-NLS-1$
+	static final String TOOLTIP = "tooltip"; //$NON-NLS-1$
 	public static final String TAB = "tab"; //$NON-NLS-1$
 	private final PropertyChangeSupport support;
-	private final Hashtable<String, Object> table;
+	private final Map<String, Object> table;
```

Comment:
If it's truly a map, why are we calling it a table?  What's it a map of?  Use that as the variable name